app/Services/VocabularyServiceProvider.js

```diff
@@ -92,10 +94,19 @@ async function getGroupVocabulary(userId, groupId) {
       },
     ],
     attributes: ['id', 'name', 'active', 'description'],
-    where: {
-      userId,
-      groupId,
-    },
+    where: search
+      ? {
+          [Op.and]: [
+            {
+              userId,
+              groupId,
+            },
+            sequelize.where(sequelize.fn('lower', sequelize.col('VocabularyCard.name')), {
+              [Op.like]: `%${search}%`,
+            }),
+          ],
+        }
+      : { userId, groupId },
```

Comment:
I think you could actually extract them like the following:
```suggestion
    where: {
        userId,
        groupId,
        ...(search ? {
            [sequelize.fn('lower', sequelize.col('VocabularyCard.name'))]: {
              [Op.like]: `%${search}%`,
            }} : {}
        )
```
*(have not tried :smile: )*

Reply:
Finde es schöner, so wie es ist

Reply:
Na dann, so haben wir halt doppelten code. Also wenn wir irgendwann weitere where filter brauchen müssen wir die zweimal hinzufügen.

Eine andere Frage, wenn du die Ergebnisse lowerst, musst du dann nicht auch den search string lowern?

Reply:
Hab beides angepasst